buildchain/buildchain/targets/remote_image.py

```diff
@@ -103,7 +103,10 @@ def task(self) -> types.TaskDict:
 
     def _skopeo_copy(self) -> List[str]:
         """Return the command line to execute skopeo copy."""
-        cmd = [config.SKOPEO, 'copy', '--format', 'v2s2']
+        cmd = [
+            config.SKOPEO, '--override-os', 'linux', '--insecure-policy',
```

Comment:
Could make that little guy configurable, since in the future we'll want to verify image signatures in CI at least.

Reply:
Then, it will be made configurable at that time :slightly_smiling_face: 

There is no point in making something configurable for "one day"™
I mean, we don't even have a ticket to track it this "verify image in CI", so I'm not expecting it to land any time soon (if ever).

Otherwise, I can also make `override-os` configurable ("one day" we may support MacOS or Windows) :stuck_out_tongue: